README.md

```diff
@@ -51,6 +51,27 @@ services:
       - INCLUDE_LOGS=true
 ```
 
+#### Running in Docker Swarm
+To run the backup service in Docker Swarm, you should configure it as a global service. This ensures that the backup process runs on each node in the Swarm and backs up all matching containers on the local node.
+```yml
```

Comment:
Nit: A lot of this YAML is duplicated from the above example. Perhaps it would be better to link out to the Docker documentation for making a service "global"?